src/Behat/Gherkin/Loader/ArrayLoader.php

```diff
@@ -289,22 +289,22 @@ private function examplesAreInArray($exHash)
      * Processes cases when examples are in the form of array of arrays
      * OR in the form of array of objects.
      *
-     * @param $exHash array hash
+     * @param $examplesHash array hash
      * @param $examplesKeyword string
      * @param $examples array
```

Comment:
those tags are invalid. The type must be before the name, not after

Reply:
Hmm, I see that in other cases, we simply used `$hash`. I think I'll do the same here.

Reply:
what I mean is that we should use `@param string $examplesKeyword`, not `@param $examplesKeyword string`. currently, `string` is not the type but the human-readable description.

Reply:
oooh, I totally missed that

Reply:
To be exact, they are not invalid as far as a phpdoc parser is concerned (the type is optional). But they are not the intended behavior (and would report a missing type in phpstan level 6 when phpstan requires knowing the type)

Reply:
Done; I also simplified/cleaned-up the code a bit as well.